### PR TITLE
change jormun instances conf from ini to json

### PR DESCRIPTION
**Pull request by antoine-de**
needed to handle realtime proxies

clean up some deprecated code in the same time

[Review on Reviewable](https://reviewable.io/reviews/canaltp/fabric_navitia/87)

don't forget to merge https://github.com/CanalTP/navitia_deployment_conf/pull/95 after

@kinnou02 @Hoshiyo can you review this ? it would be nice to test the deployment tonight. (or tomorrow)

### CODE REVIEW

**Comment by antoine-de** [comment]
@kinnou02 @Hoshiyo can you review this ? it would be nice to test the deployment tonight. (or tomorrow)

**Comment by kinnou02** [comment]
:+1:

**Comment by antoine-de** [comment]
@Hoshiyo can you take a look at those PR ? it would be great if they were merged for the dev deployment tonight

File: templates/jormungandr/instance.json.jinja [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "key": "{{instance.name}}",
+    "zmq_socket": "{{instance.jormungandr_zmq_socket_for_instance}}",
```
Comment (Hoshiyo) [inline templates/jormungandr/instance.json.jinja]: Just add comment to specify that this file is managed by fabric

Comment (antoine-de) [inline templates/jormungandr/instance.json.jinja]: we can't json does not handle comments :wink:

**Pull request merged**